ttnn/cpp/ttnn/operations/data_movement/tilize_with_val_padding/tilize_with_val_padding_pybind.hpp

```diff
@@ -53,7 +53,13 @@ void bind_tilize_with_val_padding(py::module& module) {
                bool use_multicore,
                uint8_t queue_id) {
                 return self(
-                    queue_id, input_tensor, output_tensor_shape, value, memory_config, output_dtype, use_multicore);
+                    queue_id,
+                    input_tensor,
+                    Shape(output_tensor_shape).padded_shape(),
```

Comment:
do you really have to create a Shape here?

Reply:
Not really, but I don't have an easy way to extract SimpleShape from LegacyShape, and make it visible that I'm extracting a padded_shape (not logical_shape). This conversion will go away soon, once we replace the function argument to be SimpleShape instead of LegacyShape.